src/main/java/com/revature/models/Trip.java

```diff
@@ -14,7 +15,7 @@
 @Component
 @Entity
 @Table(name = "trips")
-@NoArgsConstructor @Data
+@NoArgsConstructor @Data @Getter
```

Comment:
Getter annotation not needed. Data already includes Getter.

Reply:
Removed Getter annotation